src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/OpprettBehandlingService.kt

```diff
@@ -111,6 +119,58 @@ class OpprettBehandlingService(
         }
     }
 
+    @Transactional(readOnly = true)
+    fun kanOppretteRevurdering(fagsakId: Long): KanOppretteRevurderingResponse {
+        val fagsak = hentFagsak(fagsakId)
+        val resultat = utledKanOppretteRevurdering(fagsak)
+        return when (resultat) {
+            is KanOppretteRevurdering -> KanOppretteRevurderingResponse(true, null)
+            is KanIkkeOppretteRevurdering -> KanOppretteRevurderingResponse(false, resultat.årsak.kanIkkeOppretteRevurderingÅrsak)
+        }
+    }
+
+    @Transactional
+    fun prøvÅOppretteRevurdering(fagsakId: Long, behandlingÅrsak: BehandlingÅrsak): OpprettRevurderingResponse {
```

Comment:
Hvorfor prøvÅOppretteRevurdering?Jeg skjønner at du sjekker først om det er mulig å opprette en revurdering. Men metodenavn høres ikke bra ut, tror jeg. Kanskje validerOgOpprettRevurderingKlage er et bedre navn. Det er også viktig at vi bruker Klage på slutten. Selv om det er et generisk funksjonalitet, bruker vi ikke det på vanlig revurdering. Det kan skape forvirring for nye utvikler. Derfor er det beste å bruke klage på slutten inntil noen andre en klage bruker dette

Reply:
[prøvÅOppretteRevurdering -> validerOgOpprettRevurderingKlage](https://github.com/navikt/familie-ks-sak/pull/263/commits/d573de592b95faa6cfd08dd749385eac8ffa00b8)

Reply:
[opprettRevurdering -> opprettRevurderingKlage](https://github.com/navikt/familie-ks-sak/pull/263/commits/48b38bbaf5f69d86351bf003ea3f3e95a18d2bd6)